src/TodoCard.js

```diff
@@ -20,13 +20,13 @@ export default function TodoCard({
   setCompletedItems,
 }) {
 
-  const[disable, setDisable] = useState(0);
+  const[disable, setDisable] = useState(false);
```

Comment:
0 maynot work as a false but !1 will definitely work.

Reply:
` const[disable, setDisable] = useState(false);`

using boolean instead of 1 & 0.